docs/tutorials/multimodal/advanced_topics/presets.md

```diff
@@ -0,0 +1,124 @@
+# AutoMM Presets
+:label:`sec_automm_presets`
+
+It is well-known that we usually need to set hyperparameters before the learning process begins. Deep learning models, e.g., pretrained foundation models, can have anywhere from a few hyperparameters to a few hundred. The hyperparameters can impact training speed, final model performance, and inference latency. However, choosing the proper hyperparameters may be challenging for many users with limited expertise.
+
+In this tutorial, we will introduce the easy-to-use presets in AutoMM. Our presets can condense the complex hyperparameter setups into simple strings. More specifically, AutoMM supports three presets: `medium_quality`, `high_quality`, and `best_quality`.
+
+
+```{.python .input}
+import warnings
+warnings.filterwarnings('ignore')
+```
+
+## Dataset
+
+For demonstration, we use a subsampled Stanford Sentiment Treebank ([SST](https://nlp.stanford.edu/sentiment/)) dataset, which consists of movie reviews and their associated sentiment. 
+Given a new movie review, the goal is to predict the sentiment reflected in the text (in this case, a **binary classification**, where reviews are 
+labeled as 1 if they conveyed a positive opinion and 0 otherwise).
+To get started, let's download and prepare the dataset.
+
+
+```{.python .input}
+from autogluon.core.utils.loaders import load_pd
+train_data = load_pd.load('https://autogluon-text.s3-accelerate.amazonaws.com/glue/sst/train.parquet')
+test_data = load_pd.load('https://autogluon-text.s3-accelerate.amazonaws.com/glue/sst/dev.parquet')
+subsample_size = 1000  # subsample data for faster demo, try setting this to larger values
+train_data = train_data.sample(n=subsample_size, random_state=0)
+train_data.head(10)
+```
+
+## Medium Quality
+In some situations, we prefer fast training and inference to the prediction quality. `medium_quality` is designed for this purpose.
+Among the three presets, `medium_quality` has the smallest model size. Now let's fit the predictor using the `medium_quality` preset. Here we set a tight time budget for a quick demo.
+
+
+```{.python .input}
+from autogluon.multimodal import MultiModalPredictor
+predictor = MultiModalPredictor(label='label', eval_metric='acc', presets="medium_quality")
+predictor.fit(
+    train_data=train_data,
+    time_limit=30, # seconds
```

Comment:
If user follows this time limit on a full dataset, and the validation may not happen within the time limit, it may cause an error when we selecting the best model?

Reply:
LGTM with a minor comment